src/helpers/index.ts

```diff
@@ -0,0 +1,9 @@
+export const convertToLowerCase = (obj: any) => {
```

Comment:
@gabrielzepeda1 Let's add a type for the object passed as param on this one, please.